src/containers/chat/chat-item/ChatItem.tsx

```diff
@@ -26,29 +32,41 @@ const ChatItem: FC<ItemOfChatProps> = ({
 }) => {
   const { t } = useTranslation()
   const { userId } = useAppSelector((state) => state.appMain)
+  const userToSpeak = useMemo<Member | undefined>(
+    () => chat?.members.find((member) => member.user._id !== userId),
+    [chat, userId]
+  )
 
-  const { firstName, lastName, photo } = chat.members[0].user
-  const { text, author, updatedAt } = chat.latestMessage
+  const firstName = userToSpeak?.user?.firstName
```

Comment:
do you really need a "?" character after user, I think userToSpeak?.user.firstName will be enough

also, you can destructure user from userToSpeak instead of creating these 3 variables

Reply:
you are right, removed it